### PR TITLE
[List] CAES-1707: Fixed access to edit list.

### PR DESCRIPTION
**Pull request by aburov-4xxi**

### CODE REVIEW

**Review by beshkenadze — APPROVED** [review]

File: tests/api/Team/PermissionTest.php [hunk 1]
```diff
@@ -241,7 +241,7 @@ private function canAccessToList(User $user, Team $team)
         $I->seeResponseByJsonPathContainsJson(sprintf('$[?(@.type=="%s")]', DirectoryEnumType::DEFAULT), ['_links' => [
             'team_create_item' => [],
             'team_sort_list' => [],
-            'team_edit_list' => [],
+            //'team_edit_list' => [],
```
Comment (ikirichenko) [inline tests/api/Team/PermissionTest.php:244]: May be delete this string with comment.

File: tests/api/User/PermissionTest.php [hunk 2]
```diff
@@ -45,7 +45,7 @@ public function listPermission()
         $I->seeResponseByJsonPathContainsJson(sprintf('$[?(@.label=="%s")]', DirectoryEnumType::DEFAULT), ['_links' => [
             'sort_list' => [],
             'create_item' => [],
-            'edit_list' => [],
+            //'edit_list' => [],
```
Comment (ikirichenko) [inline tests/api/User/PermissionTest.php:48]: May be delete this string with comment.

File: tests/api/Team/PermissionTest.php (hunk 1 above)
Comment (aburov-4xxi) [reply tests/api/Team/PermissionTest.php:244]: I will revert this change after release.

File: tests/api/User/PermissionTest.php (hunk 2 above)
Comment (aburov-4xxi) [reply tests/api/User/PermissionTest.php:48]: I will revert this change after release.

**Review by ikirichenko — APPROVED** [review]

**Pull request merged**